src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx

```diff
@@ -69,6 +76,27 @@ const ActionsListItem = ({
 
   const status = ListItemStatus.Defused;
 
+  const fromDomain = safeTransaction
+    ? initiatorColony?.domains?.items[0]
+    : itemDomain;
+
+  const safeTransactionStatus = TRANSACTION_STATUS.PENDING;
+
+  const SafeTag = (
+    <Tag
+      text={SafeMSGs[safeTransactionStatus]}
+      appearance={{
+        theme:
+          safeTransactionStatus === TRANSACTION_STATUS.PENDING
+            ? 'golden'
+            : 'primary',
+        colorSchema: 'fullColor',
+      }}
+    />
+  );
+
+  const tag = safeTransaction ? SafeTag : <MotionTag />;
```

Comment:
FYI: Both tags should be shown in the case of safe transactions that are motions. You can check this issue for examples: https://github.com/JoinColony/colonyCDapp/issues/963

In the scope of this PR, making sure that the tag looks good with motion is not a concern so you can leave this in if you want. I'm just saying this so you are aware that this will be updated in the future :)

Reply:
Thanks for the future notice :eyes: